Movies/Movies/DTO/MovieDTO.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Movies.DTO
+{
+    [DataContract]
+    public class MovieDTO
+    {
+        [DataMember(Name = "id")]
+        public int Id { get; set; }
+
+        [DataMember(Name = "title")]
+        public string Title { get; set; }
+
+        [DataMember(Name = "popularity")]
+        public double Popularity { get; set; }
+
+        [DataMember(Name = "poster_path")]
```

Comment:
You can use a `SnakeCaseNamingStrategy` instead of declaring a `DataMember` for each property.